### PR TITLE
Update reward cycle length to 2000.

### LINKED ISSUE #2088: Update reward cycle length to 2000
Currently set to 1000, but should be 2000:

* https://github.com/blockstack/stacks-blockchain/blob/master/src/chainstate/stacks/boot/pox-mainnet.clar#L10
* https://github.com/blockstack/stacks-blockchain/blob/master/src/core/mod.rs#L64

### PR DESCRIPTION
**Pull request by diwakergupta**
Fixes #2088

### CODE REVIEW

**Comment by diwakergupta** [comment]
Results from `cargo test` locally -- both failures seem unrelated to the commit:

```
failures:

---- net::poll::test::test_register_deregister stdout ----
thread 'net::poll::test::test_register_deregister' panicked at 'called `Result::unwrap()` on an `Err` value: SocketError', src/net/poll.rs:520:44

---- net::poll::test::test_register_too_many_peers stdout ----
thread 'net::poll::test::test_register_too_many_peers' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 22, kind: InvalidInput, message: "Invalid argument" }', src/net/poll.rs:349:45


failures:
    net::poll::test::test_register_deregister
    net::poll::test::test_register_too_many_peers

test result: FAILED. 1147 passed; 2 failed; 87 ignored; 0 measured; 0 filtered out
```

**Review by kantai — APPROVED** [review]
Thanks for fixing this, LGTM!

**Review by jcnelson — APPROVED** [review]
LGTM!

File: src/core/mod.rs [hunk 1]
```diff
@@ -61,7 +61,7 @@ pub const CHAINSTATE_VERSION: &'static str = "23.0.0.0";
 pub const MICROSTACKS_PER_STACKS: u32 = 1_000_000;
 
 pub const POX_PREPARE_WINDOW_LENGTH: u32 = 240;
```
Comment (diwakergupta) [inline src/core/mod.rs:63]: @kantai shouldn't this match `PREPARE_CYCLE_LENGTH` in `pox-mainnet.clar` above? Happy to send a separate PR if so.
